src/util/forecast_recording_util.cpp

```diff
@@ -170,4 +170,66 @@ void ForecastRecordingUtil::RecordForecastQueryFrequencies(uint64_t timestamp_to
   }
 }
 
+void ForecastRecordingUtil::RecordAppliedAction(uint64_t timestamp_to_record, selfdriving::pilot::action_id_t action_id,
```

Comment:
This is not just forecasting recording anymore. Maybe call this `SelfDrivingRecordingUtil`.